dbms/src/Common/tests/gtest_tiflash_security.cpp

```diff
@@ -76,6 +76,24 @@ cert_allowed_cn="tidb"
     ASSERT_EQ((int)new_tiflash_config.allowedCommonNames().count("tidb"), 0);
 }
 
+TEST(TiFlashSecurityTest, EmptyConfig)
+{
+    TiFlashSecurityConfig tiflash_config;
+    const auto log = Logger::get();
+    tiflash_config.setLog(log);
+
+    String test =
+        R"(
+[security]
+ca_path=""
+cert_path=""
+key_path=""
+        )";
+    auto new_config = loadConfigFromString(test);
+    tiflash_config.update(*new_config);
+    ASSERT_FALSE(tiflash_config.hasTlsConfig());
```

Comment:
Shall we add cases for combinations of `ca_path`/`cert_path`/`key_path` each being empty/non-empty?

Reply:
The test cases have been refined